commit 73d21a27e5b4460fb9b34c51e63fdbde52d83e58
Author:     Tizen Build Automation <tizenbuild@samsung.com>
AuthorDate: Mon Oct 26 01:40:47 2026 +0000
Commit:     Tizen Build Automation <tizenbuild@samsung.com>
CommitDate: Mon Oct 26 01:40:47 2026 +0000

    Update candidate_snapshot : TIZEN/Tizen/Tizen-Base-Toolchain/project_meta,TIZEN/Tizen/Tizen-Unified-SC/project_meta,TIZEN/Tizen/Tizen-Unified-Toolchain/project_meta,TIZEN/Tizen/Tizen-Unified-X/project_meta,TIZEN/Tizen/Tizen-Unified/project_meta,TIZEN/Tizen/Tizen-Base/project_meta,TIZEN/Tizen-10.1/Tizen-10.1-Unified/project_meta,TIZEN/Tizen-10.0/Tizen-10.0-Unified/project_meta,
---
 TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot | 3 ---
 TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot | 2 --
 TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot    | 2 --
 TIZEN/Tizen/Tizen-Base/candidate_snapshot              | 1 -
 TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot        | 1 -
 TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot | 1 -
 TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot         | 2 --
 TIZEN/Tizen/Tizen-Unified/candidate_snapshot           | 3 ---
 8 files changed, 15 deletions(-)

diff --git a/TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot b/TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot
index cad7bd8..7067ece 100644
--- a/TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot
+++ b/TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
-  <candidate target="tizen-10.0-unified_20261015.031059" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
-  <candidate target="tizen-10.0-unified_20261015.031153" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
-  <candidate target="tizen-10.0-unified_20261015.092703" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
   <candidate target="tizen-10.0-unified_20261016.022219" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
   <candidate target="tizen-10.0-unified_20261019.002929" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
   <candidate target="tizen-10.0-unified_20261019.212120" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
diff --git a/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot b/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot
index ad00608..fe6ec80 100644
--- a/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot
+++ b/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
-  <candidate target="tizen-10.1-unified_20261015.031743" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
-  <candidate target="tizen-10.1-unified_20261015.092932" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261016.022138" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261018.221534" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261019.042211" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
diff --git a/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot b/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot
index ec9fee6..ee3e1ec 100644
--- a/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot
@@ -4,12 +4,10 @@
   <candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <candidate target="tizen-base-toolchain_20261015.022255" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261015.022255" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen/Tizen-Base/candidate_snapshot b/TIZEN/Tizen/Tizen-Base/candidate_snapshot
index 4e85023..eca4228 100644
--- a/TIZEN/Tizen/Tizen-Base/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base/candidate_snapshot
@@ -12,6 +12,5 @@
   <base_candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261015.022255" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot
index 18056e7..be8a5b3 100755
--- a/TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-SC/" />
-  <candidate target="tizen-unified-sc_20261015.074205" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-SC/" />
   <candidate target="tizen-unified-sc_20261019.223825" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-SC/" />
   <candidate target="tizen-unified-sc_20261021.232527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-SC/" />
   <candidate target="tizen-unified-sc_20261023.043823" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-SC/" />
diff --git a/TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot
index 1fd70a3..01b178d 100644
--- a/TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot
@@ -8,6 +8,5 @@
   <base_candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261015.022255" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot
index a9b0132..c8ef63f 100644
--- a/TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
-  <candidate target="tizen-unified-x_20261015.092750" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
-  <candidate target="tizen-unified-x_20261015.211107" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
   <candidate target="tizen-unified-x_20261018.221821" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
   <candidate target="tizen-unified-x_20261021.234914" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-X/" />
diff --git a/TIZEN/Tizen/Tizen-Unified/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified/candidate_snapshot
index 3ff195c..2aa62ff 100644
--- a/TIZEN/Tizen/Tizen-Unified/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified/candidate_snapshot
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
-  <candidate target="tizen-unified_20261015.005108" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
-  <candidate target="tizen-unified_20261015.101045" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
-  <candidate target="tizen-unified_20261015.232434" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <candidate target="tizen-unified_20261019.210549" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <candidate target="tizen-unified_20261020.133455" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <candidate target="tizen-unified_20261020.233741" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
